internal/pkg/cli/recorder/options.go

```diff
@@ -31,6 +31,7 @@ type Options struct {
 	typ            Type
 	outputFile     string
 	baseSyscalls   []string
+	noStart        bool
```

Comment:
I would name it `noProcStart` to be more clear since these are the options for recorder itself. `noStart` sounds more like not starting the recorder.